<ui version="4.0" >
 <class>Config</class>
 <widget class="QDialog" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qxygen Setup</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Background Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="bgMode" >
     <item>
      <property name="text" >
       <string>Plain</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Scanlines</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>[Reserved]</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Complex</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Vertical Top/Bottom Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Horizontal Left/Right Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Vertical Center Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Horizontal Center Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Glass</string>
      </property>
     </item>
    </widget>
   </item>
   <item rowspan="11" row="0" column="2" >
    <widget class="QTextBrowser" name="info" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Hardware Acceleration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="acceleration" >
     <item>
      <property name="text" >
       <string>oxygenPP (tiling)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Qt (QGradient)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>XRender</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>OpenGL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Checkbox Appereance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="checkMark" >
     <item>
      <property name="text" >
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>O</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Tabbar transition</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="tabTransition" >
     <item>
      <property name="text" >
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Crossfade</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Scanline blend</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Slide in</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Slide out</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Roll in</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Roll out</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Open vertically</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Close vertically</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Open horizontally</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Close horizontally</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QCheckBox" name="useGlass" >
     <property name="text" >
      <string>Use  Glass/Gloss instead of gradients on e.g. Buttons</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QCheckBox" name="inverseButtons" >
     <property name="text" >
      <string>Inverse Pushbuttons (sunken instead of raised)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QCheckBox" name="showMenuIcons" >
     <property name="text" >
      <string>Show icons on popup menus</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QCheckBox" name="menuRuphy" >
     <property name="text" >
      <string>Ruphy's menu background</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2" >
    <widget class="QCheckBox" name="menuShadow" >
     <property name="text" >
      <string>Menus have "shadow" outline</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QCheckBox" name="glassProgress" >
     <property name="text" >
      <string>Progress in a glass Tube</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Color Roles</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Buttons, Checkboxes, Radios, etc</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="crButtons" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Hovered Buttons, etc</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="crButtonsHover" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Progress Bar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="crProgressBar" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>ProgressBar Groove</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="crProgressBarGroove" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Tabbar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QComboBox" name="crTabBar" />
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Popup Menus</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QComboBox" name="crPopup" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
